Collector/RequestStackProvider.php

```diff
@@ -68,7 +68,7 @@ public static function createFromCollectedData(array $data)
 
     /**
      * @param array $messages is an array with keys 'failure', 'request' and 'response' which hold requests for each call to
-     *                        sendRequest and for each depth.
+     *                        sendRequest and for each depth
```

Comment:
Not sure if this is a solution. Normally we need that period. See https://github.com/php-http/message/pull/48